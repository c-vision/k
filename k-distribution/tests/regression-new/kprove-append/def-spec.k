module DEF-SPEC
    imports DEF
    claim <k> (a ~> _ => .) ... </k>
endmodule
