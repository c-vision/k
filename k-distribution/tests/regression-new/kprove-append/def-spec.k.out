  #Ceil ( _0 ~> _DotVar1 ~> . )
#And
  <k>
    a ~> _0 ~> _DotVar1 ~> .
  </k>
